.mailmap

```diff
@@ -0,0 +1,5 @@
+Andrew Kofink <ajkofink@gmail.com> <akofink@redhat.com>
+Bernhard Hopfenmüller <hopfenmueller@atix.de> <bernhardh@atix.de>
+Christoffer Reijer <christoffer.reijer@basalt.se> <ephracis@gmail.com>
+Eric D. Helms <ericdhelms@gmail.com> <eric.d.helms@gmail.com>
+Matthias Dellweg <dellweg@atix.de>
```

Comment:
@mdellweg you're just here because you sometimes commit as "mdellweg" and sometimes as "Matthias Dellweg", which confuses shortlog w/o that entry